[2024-06-05T16:36:11.313+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T16:36:11.360+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T16:36:11.374+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T16:36:11.375+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-05T16:36:11.402+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_insert_data> on 2024-06-04 00:00:00+00:00
[2024-06-05T16:36:11.418+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=65) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T16:36:11.423+0000] {standard_task_runner.py:63} INFO - Started process 68 to run task
[2024-06-05T16:36:11.419+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_postgres', 'download_and_insert_data', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '704', '--raw', '--subdir', 'DAGS_FOLDER/pipe.py', '--cfg-path', '/tmp/tmpsmzrq1th']
[2024-06-05T16:36:11.425+0000] {standard_task_runner.py:91} INFO - Job 704: Subtask download_and_insert_data
[2024-06-05T16:36:11.667+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [running]> on host 3ca4815e3382
[2024-06-05T16:36:11.897+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_postgres' AIRFLOW_CTX_TASK_ID='download_and_insert_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-06-05T16:36:11.903+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T16:36:15.614+0000] {python.py:237} INFO - Done. Returned value was: /tmp/data_1000.csv
[2024-06-05T16:36:15.616+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T16:36:15.685+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_postgres, task_id=download_and_insert_data, run_id=scheduled__2024-06-04T00:00:00+00:00, execution_date=20240604T000000, start_date=20240605T163611, end_date=20240605T163615
[2024-06-05T16:36:15.754+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T16:36:15.808+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-05T16:36:15.812+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-05T17:16:53.534+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T17:16:53.562+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T17:16:53.571+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T17:16:53.572+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-05T17:16:53.588+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_insert_data> on 2024-06-04 00:00:00+00:00
[2024-06-05T17:16:53.595+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=79) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T17:16:53.598+0000] {standard_task_runner.py:63} INFO - Started process 82 to run task
[2024-06-05T17:16:53.597+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_postgres', 'download_and_insert_data', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '737', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpw8_c261c']
[2024-06-05T17:16:53.599+0000] {standard_task_runner.py:91} INFO - Job 737: Subtask download_and_insert_data
[2024-06-05T17:16:53.659+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_postgres.download_and_insert_data scheduled__2024-06-04T00:00:00+00:00 [running]> on host 68d2d1c90970
[2024-06-05T17:16:53.762+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_postgres' AIRFLOW_CTX_TASK_ID='download_and_insert_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-06-05T17:16:53.763+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T17:16:57.660+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-06-05T17:16:59.603+0000] {logging_mixin.py:188} INFO - Data insertion complete
[2024-06-05T17:16:59.605+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T17:16:59.606+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T17:16:59.615+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_postgres, task_id=download_and_insert_data, run_id=scheduled__2024-06-04T00:00:00+00:00, execution_date=20240604T000000, start_date=20240605T171653, end_date=20240605T171659
[2024-06-05T17:16:59.643+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T17:16:59.678+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-05T17:16:59.681+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
